bindings/java/c/evmc-vm.c

```diff
@@ -78,9 +90,10 @@ JNIEXPORT void JNICALL Java_org_ethereum_evmc_EvmcVm_execute(JNIEnv* jenv,
                                                              jint jrev,
                                                              jobject jmsg,
                                                              jobject jcode,
-                                                             jint jsize,
+                                                             jint jcode_size,
                                                              jobject jresult)
 {
+    (void)jcls;
     struct evmc_message* cmsg = (struct evmc_message*)(*jenv)->GetDirectBufferAddress(jenv, jmsg);
     assert(cmsg != NULL);
     const uint8_t* ccode = (uint8_t*)(*jenv)->GetDirectBufferAddress(jenv, jcode);
```

Comment:
Maybe you want to remove the `c` prefixes from these things. And drop the large debugging comment?

Reply:
Why to drop debugging section?

Reply:
Is it useful? I don't mind.